website/www/site/content/en/get-started/downloads.md

```diff
@@ -88,6 +88,14 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.39.0 (2022-05-25)
+Official [source code download](https://www.apache.org/dyn/closer.cgi/beam/2.39.0/apache-beam-2.39.0-source-release.zip).
+[SHA-512](https://downloads.apache.org/beam/2.39.0/apache-beam-2.39.0-source-release.zip.sha512).
+[signature](https://downloads.apache.org/beam/2.39.0/apache-beam-2.39.0-source-release.zip.asc).
+
+[Release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12351169)
+[Blog post](/blog/beam-2.39.0).
+
 ### 2.38.0 (2022-04-20)
 Official [source code download](https://www.apache.org/dyn/closer.cgi/beam/2.38.0/apache-beam-2.38.0-source-release.zip).
```

Comment:
Heads up, you also need to update the URLs for 2.38.0 to use the archive, like 2.37.0. (For example: https://github.com/apache/beam/pull/17378/files#diff-967d930b8b810282f4eddbf2ecf6cf701b5d377be7cb26a2680bd30649678933)

Reply:
Was this done?

Reply:
yes